integration-tests/tests/test_cases.py

```diff
@@ -114,6 +119,43 @@ def setup_method(self):
         except (KeyError, json.JSONDecodeError):
             raise Exception(response.content)
 
+class EthApiTestCase(TransactionTestCase):
+    def get_block_header(self, block_number):
+        method = "ethclient_headerByNumber"
+        params = [self.network_id, block_number]
+        return self.rpc_valid_request(method, params, 13)
+
+    def get_transaction_receipt(self, tx_hash):
+        method = "ethclient_transactionReceipt"
+        params = [self.network_id, tx_hash]
+        return self.rpc_valid_request(method, params, 13)
```

Comment:
i am lame, what is 13 here?

Reply:
some sort of ID thingy to know who's making the RPC request, AFAIU
I've just seen 13 is used as a default value, so I can just skip the parameter.